Dockerfile

```diff
@@ -19,12 +20,15 @@ ARG IMAGE_REPO_PREFIX
 ENV GITCOMMIT=$GITCOMMIT VERSION=$VERSION BUILDTIME=$BUILDTIME IMAGE_REPO_PREFIX=$IMAGE_REPO_PREFIX
 ENV CGO_ENABLED=0
 RUN make bin/compose-controller bin/compose-controller.test e2e-binary bin/installer bin/api-server bin/api-server.test
+RUN go get github.com/onsi/ginkgo/ginkgo
```

Comment:
Can we put a fixed release here?

Reply:
Ginkgo does not vendor its dependencies... So it seems unlikely to be doable